library/stdtypes.po

```diff
@@ -403,9 +408,9 @@ msgstr ""
 "complexes ont une partie réelle et une partie imaginaire, qui sont chacune "
 "des nombres à virgule flottante. Pour extraire ces parties d'un nombre "
 "complexe *z*, utilisez ``z.real`` et ``z.imag``. (La bibliothèque standard "
-"comprend d'autres types numériques, :mod:`fractions` qui stocke des "
-"rationnels et :mod:`decimal` qui stocke les nombres à virgule flottante avec "
-"une précision définissable par l'utilisateur.)"
+"comprend d'autres types numériques, :mod:`fractions.Fraction` qui stocke des "
```

Comment:
Un petit message de padpo sur un item que tu as modifié, mais pas dans ce que tu as fait :

/tmp/padpo_j77_iede/library/stdtypes.po: 386 [     NBSP     ] There should be a non-breakable space before ":": between ###rois types numériques distincts### and ###: « integers » (entiers), « flo###